<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>30</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>331</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 80px;
background-color: rgb(255, 255, 127);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string>Se connecter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>251</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);

border:1.5px solid ;
border-color: rgb(0, 85, 0);
color:rgba(0,0,0,240);
padding-bottom:3.5px;</string>
     </property>
     <property name="placeholderText">
      <string>Nom d'utilisateur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>251</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);

border:1.5px solid ;
border-color: rgb(0, 85, 0);
color:rgba(0,0,0,240);
padding-bottom:3.5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Mot de passe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>410</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0, 0, 0, 0.464);</string>
     </property>
     <property name="text">
      <string>Veuillez saisir votre identifiant et mot de passe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(0, 85, 0););
	color:rgb(0, 85, 0);
	border-radius:10px;
}

QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0  rgb(0, 85, 0), stop:1 rgb(0, 85, 0););
}

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0  rgb(0, 85, 0), stop:1 rgba(11, 131, 120, 219));
}
</string>
     </property>
     <property name="text">
      <string>Se connecter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>521</width>
       <height>491</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/cosmetcis_image/da09915c774a24097eeab7d3fa22af48.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>440</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0, 0, 0, 0.464);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>label_4</zorder>
    <zorder>pushButton</zorder>
    <zorder>label</zorder>
    <zorder>label_6</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
